dffml/source/file.py

```diff
@@ -66,10 +77,19 @@ def __repr__(self):
             elif self.filename[::-1].startswith(('.xz')[::-1]) or \
                     self.filename[::-1].startswith(('.lzma')[::-1]):
                 close = lzma.open(self.filename, 'wt')
+            elif zipfile.is_zipfile(self.filename):
+                tmp = tempfile.NamedTemporaryFile(suffix='.csv')
```

Comment:
lets avoid the use of a tempfile here. I think a helper method like with load might be useful, see https://docs.python.org/3/library/zipfile.html#zipfile.ZipFile.write or maybe https://docs.python.org/3/library/zipfile.html#zipfile.ZipFile.writestr

Reply:
I have added a helper method similar to the one in _open method.